librust/proc_macro/rust/bridge/ident.rs

```diff
@@ -0,0 +1,59 @@
+use bridge::span::Span;
+use std::ffi::{c_char, CStr, CString};
```

Comment:
We can think about this later and this is fine for now, but eventually it'd be nice to try and stay away from `std` as much as possible for this crate. I think it'll still be some time before `gccrs` can compile `std`.

Reply:
We will need `std` anyway, some types/functions/trait come from the standard library. Mainly `fmt`.